core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/plangen/LogicalPlan.java

```diff
@@ -1,5 +1,6 @@
 package edu.uci.ics.texera.dataflow.plangen;
 
+import edu.uci.ics.texera.api.engine.MultipleSinkPlan;
```

Comment:
move it to line 15 so that similar imports are grouped together, (for clarity purposes)

Reply:
I will move it.